### PR TITLE
BZ#1404514: Added new sections to the OSP+CFME quickstart

### PR DESCRIPTION
**Pull request by ddomingorh**
...and in the process, also single-sourcing bits for:

- storage management
- topology widget intro
- creating custom buttons
ETC

### CODE REVIEW

File: doc-QuickStart_OpenStack_Provider_Guide/cfme/master.adoc [hunk 1]
```diff
@@ -59,15 +89,56 @@ include::topics/managing-policies.adoc[]
 
 Cloud instance provision goes through three phases:
```
Comment (dayleparker) [inline doc-QuickStart_OpenStack_Provider_Guide/cfme/master.adoc]: Should this be "Cloud instance provisioning"?

File: doc-QuickStart_OpenStack_Provider_Guide/cfme/master.adoc [hunk 2]
```diff
@@ -7,12 +7,28 @@ OpenStack Documentation Team <rhos-docs@redhat.com>
...
+* The {product-title_short} *Topology* widget (xref:using-the-topology-widget[]) provides an interactive visualization of the OpenStack cloud. 
+* {product-title_short} provides a user interface for managing OpenStack storage resources (xref:storage_managers[]).
+* _Custom buttons_, which allows you to provide automation for specific actions to OpenStack tenants (xref:custom-button-tenants[]).
+
+When adding an OpenStack cloud provider, you can also:
+
+* Enable _tenant mapping_. This creates a one-to-one association between tenants in {product-title_short} and OpenStack.
+* Connect to OpenStack through the Keystone V3 API. This API enables multiple OpenStack identity domains. Domains are high-level containers for projects, users, and groups. Users of different domains can be represented in different authentication back ends.
+
+For information about tenant mapping and the Keystone V3 API, see xref:add-openstack-oc[]
```
Comment (dayleparker) [inline doc-QuickStart_OpenStack_Provider_Guide/cfme/master.adoc]: Add full stop.

**Comment by dayleparker** [comment]
Hi Don @ddomingorh ,
The changes look great. I found a few very minor things in my review (listed in https://bugzilla.redhat.com/show_bug.cgi?id=1404514#c4) -- would you mind having a look and fixing them if needed? Please let me know when you're ready for me to merge the content with these last edits.
Thanks so much,
Dayle

**Comment by ddomingorh** [comment]
Thanks @dayleparker, I applied the corrections you mentioned. Should be ready to merge.

**Comment by dayleparker** [comment]
Awesome, thanks @ddomingorh -- all looks great to me. Merging :)

**Pull request merged**